README.md

```diff
@@ -4,4 +4,3 @@ This repo contains the source for DORA's public-facing knowledge site: https://d
 [Hugo](https://gohugo.io), a static site generator, is used to transform the contents of `/content` into HTML/CSS/JS, which is hosted via Firebase.
 
 See `/docs` for technical documentation.
-
```

Comment:
Actually, I really like those linebreaks. In fact, I'd like there to be three of them.